src/components/Editor/DrawingBoard/Canvas/CanvasWrapper.ts

```diff
@@ -8,6 +8,10 @@ export default class CanvasWrapper {
 
   private height = 0;
 
+  private styleWidth = 0;
```

Comment:
Is there any reason to add `styleWidth` and `styleHeight`? Please add a comment.

Reply:
Thank you for your comment. it would be better to add it as you said 
I'm going to add it😄